css/components/widgets/widget_list.scss

```diff
@@ -21,11 +21,30 @@
       &.-grid {
         .c-widget-card {
           transition: box-shadow .3s, transform .3s;
-          
+
```

Comment:
Set a padding right to titles to avoid this
![image](https://user-images.githubusercontent.com/8928965/35149530-b541b79a-fd16-11e7-8010-3dfb8825edd1.png)

Reply:
I would prefer to do a proper revamp of the cards, but yeah.. Let's go with this by now.